README.md

```diff
@@ -34,11 +46,40 @@ By default the build script runs an environment check, builds the image and then
 ```
 
 ## Adding update or setup routines to the toolbox
-During the initramfs routine (executed between 2st stage U-boot and 3th stage Linux debian load) this toolbox image mounted and the executable entryscript '/sbin/startup.sh' will be executed. This script shall be used to execute any required routine. Very important here is to make sure all required resources are nicely cleaned up and errors are properly handled, not to bring the system in an unrecoverable state. 
+During the initramfs routine (executed between 2st stage U-boot and 3th stage Linux debian load) 
+this toolbox image mounted and the executable entryscript '/sbin/start_update.sh' will be executed.
+ 
+The 'system configuration' scripts are installed in /etc/system_update.d folder and are prefixed 
+with a two digit number so the 'start_update' script can sort them.
+ 
+A chrooted environment will be prepared with the 'sysroot' files of the 'um-update_toolbox' files,
+all scripts use te same runtime environment in which the required variables are defined.   
+
+#### Important environment variables 
+
+* TARGET_STORAGE_DEVICE    - The device to perform the system configuration on
+* PARTITION_TABLE_FILE     - The partition table file used in the prepare_disk script,
+                             default is 'jedi_emmc_sfdisk.table', implicitly the 
+                             script will look for a counterpart checksum file:
+                             '<filename>.sha512'
+* UPDATE_EXCLUDE_LIST_FILE - A file containing all files and directories to be excluded
+                             used by the 'update_files script', default is 'jedi_update_exclude_list.txt'
+* UPDATE_ROOTFS_SOURCE     - This is the directory within the 'toolbox sysroot' that has the update files,
+                             this is used in the update_files script as the source directory for the
+                             update files. 
+* SYSTEM_UPDATE_DIR        - The system update directory is a directory within the 'toolbox sysroot' 
+                             that contains the system configuration scripts. The configuration files 
+                             can be found in '/etc/system_update', i.e. the partition table and exclude
+                             list file. The system configuration scripts can be found in the same 
+                             directory post-fixed with '.d' (/etc/system_update.d). 
```

Comment:
It looks like the variables should be somewhere at the top, and then referenced int he rest of the document.

e.g. you now reference the /etc/system_update directory 2 or 3 times, whereas instead, it would have been nice to refer to $SYSTEM_UPDATE_DIR instead